CODEOWNERS

```diff
@@ -1,3 +1,8 @@
+<!--
+SPDX-FileCopyrightText: 2022 Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
```

Comment:
Looks like you need to use # for the header.

Reply:
Yeah, I noticed that one, and there are a few others that probably are not in the right format either. I will run some tests before the next push. Thanks for pointing the issue.